### ISSUE TITLE
Remove useless invocation of Helmet in App

### ISSUE THREAD

**Issue by fzzzy** [mozilla/testpilot#3572]
We're using Helmet in App/index.js to render a static string. It's a no-op because the title was already set to that in the static html. All Helmet is doing is increase the bundle size. We should either just remove Hemlet, or make it work with fluent-react to actually localize the title, which was the intended use.

**Comment by johngruen** [comment]
AFAIK helmet is changing page titles for experiment pages

**Comment by fzzzy** [comment]
Yep, you’re right. First it sets it to Firefox Test Pilot in App, which is unnecessary, then the experiment page changes it again.

**Comment by johngruen** [comment]
Ah, got it
